scripts/configure-venue-access.ts

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env node -r esm -r ts-node/register
+
+import fs from "fs";
+import admin from "firebase-admin";
+import "firebase/firestore";
+
+import { VenueAccessMode } from "../src/types/VenueAcccess";
+
+import { initFirebaseAdminApp } from "./lib/helpers";
+
+function usage() {
+  console.log(`
+${process.argv[1]}: Configure venue access. Supports configuring a secret password, list of emails which can access the venue, or ticket codes.
+
+Usage: npx ts-node ${process.argv[1]} PROJECT_ID VENUE_ID [password|emaillist|codelist] [password | emails file path | codes file path]
+
+Example: npx ts-node ${process.argv[1]} co-reality-map password abc123
+Example: npx ts-node ${process.argv[1]} co-reality-map emails emails-one-per-line.txt
+Example: npx ts-node ${process.argv[1]} co-reality-map codes ticket-codes-one-per-line.txt
+`);
+  process.exit(1);
+}
```

Comment:
[nit] You could use the `makeScriptUsage` helper here. Example:

- https://github.com/sparkletown/sparkle/blob/staging/scripts/get-badges.ts#L16-L22

Reply:
We'll probably also want to be able to specify a service key, so that it can be used against all of the environments. Example code:

- https://github.com/sparkletown/sparkle/blob/staging/scripts/get-badges.ts#L24-L29
- https://github.com/sparkletown/sparkle/blob/staging/scripts/get-badges.ts#L42-L46